src/koi-tools.cpp

```diff
@@ -0,0 +1,50 @@
+#include "koi-tools.h"
+
+#include <iostream>
+#include <dlfcn.h>
+#include <MGConfItem>
+
+#include <QTime>
+#include <QDate>
+#include <QFile>
+
+int AsteroidOS::LCD_Tools::KoiSyncTime() {
+	char outputBuff[7] = {0xFE,0x01,0x01,0x00,0x00,0x00,0x00};
+	outputBuff[3] = (uint8_t) !MGConfItem("/org/asteroidos/settings/use-12h-format").value().toBool();
+	Write(outputBuff);
+
+	QDate dateNow = QDate::currentDate();
+	QTime timeNow = QTime::currentTime();
+	outputBuff[0] = (uint8_t) 0x50;
+	outputBuff[1] = (uint8_t) 0;
+	outputBuff[2] = (uint8_t)((((dateNow.year() - 2000) & 0xEF) << 1) | ((dateNow.month() & 0x08) >> 3));
+	outputBuff[3] = (uint8_t) ((dateNow.month() & 0x07) << 5) | ((dateNow.day() & 0x1F));
+	outputBuff[4] = (uint8_t) (((dateNow.dayOfWeek() & 0x07) << 5) |  (timeNow.hour() & 0x1F));
+	outputBuff[5] = (uint8_t) (timeNow.minute() & 0xFF);
+	outputBuff[6] = (uint8_t) ((timeNow.second() + 1) & 0xFF); //the original casio code seems to add one second here to compensate for an update delay in the subcpu
+	return Write(outputBuff);
+
+}
+
+int AsteroidOS::LCD_Tools::KoiSetDisplayColor(bool value) {
+	char outputBuff[7] = {0xFE,0x01,0x05,0x00,0x00,0x00,0x00};
+	outputBuff[3] = (uint8_t) value;
+	return Write(outputBuff);
+}
+
+int AsteroidOS::LCD_Tools::KoiPrepareTimepiece() {
+	char outputBuff[7] = {0x02,0xC1,0xBE,0x78,0x3F,0x91,0xC7};
+	return Write(outputBuff);
+}
+
+int AsteroidOS::LCD_Tools::Write(char* data) {
+	QFile mMSensorFile("/dev/MultiSensors_CD_01");
```

Comment:
the 'm' prefix is a convention that means "member", it is meant to denote class members. In this case you have a stack variable so multiSensorFile sounds like a more appropriate name

Reply:
the mMsensors file isn't appropriate, that's an accidental holdover from the narwhal qml module which I used as an example. 
I hope there's nothing wrong with mixing upper CamelCase (which lecris seems to have used initially) and lower camelCase (which seems to be more common in Qt context) here